### ISSUE TITLE
Current Breadcrumb shouldn't be able to have the focus ring style

### ISSUE THREAD

**Issue by snowystinger** [adobe/react-spectrum#1962]
# 🐛 Bug Report 

The CSS changes in this previous PR should fix the issue, we just need to make a PR with them alone as we had to revert this one for a little bit.
https://github.com/adobe/react-spectrum/pull/728
This should be testable in our docs if you go to a Collection component (such as ListBox), drill down into the props so there are a few breadcrumbs, keyboard navigate to an early breadcrumb and press 'Enter', then tab elsewhere, the focus ring should disappear as soon as the breadcrumb becomes the current one.

## 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| react-spectrum   |
| Browser          |
| Operating System |

**Comment by snowystinger** [comment]
Closing as this was only needed because the docs fix for breadcrumb focus relied on focusing the last element
